model/logs/mobile-crashes.yaml

```diff
@@ -0,0 +1,51 @@
+groups:
+  - id: mobile-crash-event
+    type: event
+    brief: >
+      This document defines attributes for crashes represented using Log Record Events.
+      `event.domain` is 'device' and `event.name` is 'crash
+    attributes:
+      - id: platform
+        requirement_level: "required"
+        brief: >
+            This attribute indicates the operating system name of the device on which the crash event occurred.
+        type:
+          allow_custom_values: false
+          members:
+            - id: android
+              value: 'android'
+              brief: 'Value if crash happened on an android device.'
+            - id: ios
+              value: 'ios'
+              brief: 'Value if crash happened on an ios device.'
+      - id: foreground
```

Comment:
why not reusing the `ios.state` and `android.state` here instead?
https://github.com/open-telemetry/semantic-conventions/blob/main/docs/mobile/events.md